.github/workflows/release.yml

```diff
@@ -7,6 +7,7 @@ on:
 jobs:
   createReleasePR:
     name: createReleasePR
+    runs-on: ubuntu-latest
```

Comment:
this doesn't seem like it's in the right PR.

Reply:
true, but it's on develop I think (should be there)